src/V3PreLex.h

```diff
@@ -170,6 +174,9 @@ class V3PreLex final {
     bool m_defQuote = false;  // Definition value inside quote
     string m_defValue;  // Definition value being built.
     int m_enterExit = 0;  // For VL_LINE, the enter/exit level
+    int m_protLength;  // unencoded length for BASE64
```

Comment:
```suggestion
    int m_protLength = 0;  // unencoded length for BASE64
```

Not necessary but harmess and prevents future trouble